src/Core/Resolvers/DWSqlQueryBuilder.cs

```diff
@@ -209,7 +209,8 @@ public string Build(SqlDeleteStructure structure)
         /// <inheritdoc />
         public string Build(SqlExecuteStructure structure)
         {
-            throw new NotImplementedException("DataWarehouse sql currently does not support executes");
```

Comment:
Are we able to use Azure Synapse SQL for integration tests?

Reply:
This is something we have to follow up on with azure synapse sql team to actually provision the db in the test pipelines and test against it. I have added the integration tests for dwsql for this pr that work on mssql instance.